modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -25,7 +25,7 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::validParams()
   params.addParam<MaterialPropertyName>("environmental_factor",
                                         "Optional coupled environmental factor");
 
-  MooseEnum error_limit_behavior("ERROR WARN IGNORE", "ERROR");
+  MooseEnum error_limit_behavior("ERROR=0 EXCEPTION=1 WARN=2 IGNORE=3", "EXCEPTION");
```

Comment:
why numbers? 👍  to default change.

Reply:
Its because we are using the same windowFailure enum for error_limit and extrapolated_limit but error_limit has different valid enums than extrapolated_limit.  So to make extroplated_limit and error_limit correspond to the correct windowFailure enum, I needed the numbers.

Reply:
Oh interesting, did that really screw things up beforehand?

Reply:
They were fine until I added extrapolate.  That messed up the indexing in the extrapolatedLimitBehavior,  so instead of getting a warning, it would give an exception because that is the second entry in the window failure enum that is corresponding to the 2nd entry in the extrapolated limit behavior.  For how we are using it, the string doesn't matter.